terraform/outputs.tf

```diff
@@ -0,0 +1,18 @@
+output "app_name" {
+  value = juju_application.dex_auth.name
+}
+
+output "provides" {
+  value = {
+    dex_oidc_config   = "dex-oidc-config",
+    grafana_dashboard = "grafana-dashboard",
+    metrics_endpoint  = "metrics-endpoint",
+  }
+}
+
+output "requires" {
+  value = {
+    ingress     = "ingress",
+    oidc_client = "oidc-client",
```

Comment:
what about [logging](https://github.com/canonical/dex-auth-operator/blob/main/metadata.yaml#L76)?

Reply:
Was this intentional? Because I see that you mention that the module is compatible with 1.8? I think we should say that this is from 1.9 and also mention logging here. If we need modules for 1.8, then that should be separate work.